drake/systems/Simulation.h

```diff
@@ -74,11 +111,24 @@ void simulate(const System& sys, double ti, double tf,
       Eigen::VectorXd::Zero(getNumInputs(sys)));
   typename System::template OutputVector<double> y;
 
+  bool rt_warning_printed = false;
+
   // Take steps from ti to tf.
   double t = ti;
   while (t < tf) {
-    handle_realtime_factor(start, t, options.realtime_factor,
-                           options.timeout_seconds);
+    try {
+      handle_realtime_factor(start, t, options.realtime_factor,
+                             options.timeout_seconds);
+    } catch (std::runtime_error err) {
```

Comment:
Catch by-reference
http://stackoverflow.com/questions/2023032/catch-exception-by-pointer-in-c#answer-2023045